src/test/java/com/contrastsecurity/sdk/scan/CodeArtifactsPactTest.java

```diff
@@ -65,6 +68,10 @@ void before(@TempDir final Path tmp) throws IOException {
     try (JarOutputStream jos = new JarOutputStream(new FileOutputStream(jar.toFile()))) {
       jos.putNextEntry(new ZipEntry("HelloWorld.class"));
     }
+    metadataJson = tmp.resolve("prescan.json");
+    try (FileOutputStream fos = new FileOutputStream(metadataJson.toFile())) {
```

Comment:
Typically, when you're writing character data, you want to wrap that `FileOutputStream` with an `OutputStreamWriter` and have the writer take care of the character encoding, but since this is just a small test, the `"literal".getBytes(StandardCharsets.UTF_8)` approach is probably fine.

Reply:
thanks for the idiomatic pattern, still learning to be a Java dev :).  I changed to match the suggestion.